FRAuth/FRAuth/DeviceBinding/UserDeviceKeyService.swift

```diff
@@ -20,45 +20,41 @@ public protocol UserKeyService {
     
     
     /// Get all the user keys in device.
-    var userKeys: [UserKey] { get set }
+    func getAll() -> [UserKey]
+    
+    
+    /// Delete user key
+    func delete(userKey: UserKey)
 }
 
 
 internal class UserDeviceKeyService: UserKeyService {
     private var deviceRepository: DeviceRepository
```

Comment:
change this to DeviceBindingRepository

Reply:
The name changes will be implemented in [SDKS-2347](https://bugster.forgerock.org/jira/browse/SDKS-2347) .